src/components/Organisms/TodoCard/index.jsx

```diff
@@ -1,11 +1,20 @@
-import React from "react";
-import { useState } from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
 import AddTaskButton from "../../Atoms/AddTaskButton";
 import Task from "../../Molecules/Task";
 import COLOR from "../../../variables/color";
+
+const localStorageKey = "taskList";
```

Comment:
```suggestion
const LOCAL_STORAGE_KEY = "taskList";
```
動的に変更されない定数に関しては、大文字の名前にするのが一般的です。